examples/methods/plot_dasvm.py

```diff
@@ -0,0 +1,231 @@
+"""
+Plot for the dasvm estimator
+======================
+
+This example illustrates the dsvm method from [21].
+
+"""
+
+
+# Author: Ruben Bueno
+#
+# License: BSD 3-Clause
+# sphinx_gallery_thumbnail_number = 2
+
+
+# %% Imports
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.lines as mlines
+
+from skada._self_labeling import DASVMClassifier
+from skada.datasets import make_dataset_from_moons_distribution
+from skada import source_target_split
+
+from sklearn.base import clone
+
+from sklearn.svm import SVC
+
+
+RANDOM_SEED = 42
+
+# base_estimator can be any classifier equipped with `decision_function` such as:
+# SVC(kernel='poly'), SVC(kernel='linear'), LogisticRegression(random_state=0), etc...
+# however the estimator has been created only for SVC
+base_estimator = SVC()
+
+target_marker = "s"
+source_marker = "o"
+xlim = (-1.5, 2.4)
+ylim = (-1, 1.3)
+
+# %%
+# We generate our dataset
+# ------------------------------------------
+#
+# We generate a simple 2D covariate shift dataset.
```

Comment:
We generate a simple 2D  dataset from moon distributionwhere source and target are not taken from the same location in the moons. Something like that